tests/test_semantic_segmentation.py

```diff
@@ -0,0 +1,23 @@
+"""Unit test related to semantic segmentation model instanciation
```

Comment:
Unit test related to the semantic segmentation model.

Reply:
Done.